kvdb/etcd_test.go

```diff
@@ -102,15 +102,6 @@ func TestEtcd(t *testing.T) {
 			name: "bucket sequence",
 			test: testBucketSequence,
 		},
-		{
-			name: "key clash",
-			test: testKeyClash,
```

Comment:
Can't this test stay now with `etcdDebug` enabled?

Reply:
Agree, added back. 
Kind of ugly because we need to duplicate the `debug.go/nodebug.go` in `kvdb` too to avoid introducing circular dependency.